workflows/Utility/modules/LogCombiner/workflow/Snakefile

```diff
@@ -0,0 +1,65 @@
+"""LogCombiner-BEAST1
+
+This module takes two log or tree files from BEAST (v1.10.4/5) as input and returns a single combined log or tree file as output. Please see https://beast.community/logcombiner for more information.
+
+Params:
+    trees (boolean): use this option to combine tree log files (default is false)
+    decimal (boolean): this option converts numbers from scientific to decimal notation (default is false)
+    burnin (integer): number of states to be considered as 'burn-in' (default is 100000)
+    resample (integer): resample the log files to this frequency (the original sampling frequency must be a factor of this value) (default is 10000)
+    scale (float): a scaling factor that will multiply any time units by this value
+    strip (boolean): strip out all annotations (trees only)
+    renumber (boolean): this option renumbers output states consecutively (default is false)
+    input:
```

Comment:
Just noticed this - 'input' doesn't exist in the config file (infile and outfile are flat with the other parameters).